### ISSUE TITLE
Publish new `analyzer` package.

### ISSUE THREAD

**Issue by pq** [dart-lang/sdk#29670]
The currently published package constrains downstream deps to an old version of `cli_util`.  As of https://github.com/dart-lang/sdk/commit/ba1a6e39f3d0892052e0e8cb4cdcd364d683ed45 this got bumped to the latest (`^0.1.0`).  It would be awesome to push a new version so the linter can play with the new logging bits in 0.1.0. 👍 

Thanks!

cc @bwilkerson

**Comment by bwilkerson** [comment]
Yep. I heard about this earlier today. We need to determine whether there have been any breaking changes since the last published version. If there are, I think we'll want to publish without publishing the breaking changes, given how recent the last set of breaking changes were.

**Comment by pq** [comment]
Fair enough.  It'd be awfully easy to push a version w/ just that change.  Maybe we should just go that route to be safe?

**Comment by bwilkerson** [comment]
That would be fine with me.

**Comment by pq** [comment]
My plan is to cherry-pick this change onto https://github.com/dart-lang/sdk/commit/ba1a6e39f3d0892052e0e8cb4cdcd364d683ed45.  I wasn't planning to create a branch (we haven't consistently), but could.  Either way, how does `0.30.1` sound for a version number?

@bwilkerson @devoncarew

**Comment by bwilkerson** [comment]
sgtm

**Comment by pq** [comment]
Published `0.30.0+1` based on ba1a6e3.

pubspec update to HEAD up for review: https://codereview.chromium.org/2895823002/

**Comment by pq** [comment]
Hmmmm.  Consuming this package from the linter is giving me fits now:

```
Failed to precompile test:test:
Unhandled exception:
Could not import "package:front_end/src/base/api_signature.dart" from "package:analyzer/src/generated/engine.dart": FileSystemException: Cannot open file, path = '/Users/pquitslund/.pub-cache/hosted/pub.dartlang.org/front_end-0.1.0-alpha.4/lib/src/base/api_signature.dart' (OS Error: No such file or directory, errno = 2)
null
```

May push a quick +2 to get us back to sanity and try again on Monday...

**Comment by pq** [comment]
OK, so this'll learn me for doing things on a Saturday night.   I'm having a hard time figuring out what commit corresponds to 0.30.0...

I see https://github.com/dart-lang/sdk/issues/29516, but I'm not sure what the last commit was.

**Comment by pq** [comment]
Sorry for all the Saturday night noise.  FWIW I think things are properly sorted now.  

For the curious, I resorted to downloading the archive for `0.30.0` from pub, modifying it publishing from there.  (I'm a little embarrassed about this so please, if there was a better way I'm all ears.)

I've got a new PR up to get the pubspec in HEAD up to date: https://codereview.chromium.org/2897783002/.

I think once that lands, sanity is restored and clients can go ahead an enjoy the new goodies in `cli_util`. :)

**Comment by pq** [comment]
Optimistically closing as `0.30.0+2` is up and I've confirmed the linter can consume it. 👍
